.github/labeler.yml

```diff
@@ -4,3 +4,18 @@ documentation:
 explorer:
   - changed-files:
       - any-glob-to-any-file: apps/explorer/**/*
+wallet:
+  - changed-files:
+      - any-glob-to-any-file: apps/wallet/**/*
+wallet-dashboard:
+  - changed-files:
+      - any-glob-to-any-file: apps/wallet-dashboard/**/*
+ts-sdk:
+  - changed-files:
+      - any-glob-to-any-file: sdk/typescript/**/*
+apps-ui-kit:
```

Comment:
Should we go `ui-kit` here for consistency ? :)

Reply:
`apps-ui-kit` is fine imo